rbac/internal/views.py

```diff
@@ -637,44 +637,51 @@ def reset_imported_tenants(request: HttpRequest) -> HttpResponse:
     # Request should accept a query parameter to exclude certain tenants so we can exclude the ~129
     excluded = request.GET.getlist("exclude_id", [])
 
+    # The default query created with "ready=false" flag otherwise is used query that checks that tenant
+    # does not have records in all tables.
+    query_with_ready_flag = request.GET.get("query_with_ready_flag", "true").strip().lower() == "true"
```

Comment:
maybe we should call it query_with_only_ready_flag since we are excluding other query conditions

Reply:
yeah the naming here is the most difficult part ... so the new query is only with `ready=false` flag and the original query is wihout `ready` flag but with all the conditions that search for related objects

Reply:
maybe we should call it `only_ready_false_flag` or something like that 🤔

Reply:
sounds good 👍

Reply:
renamed